### PR TITLE
remove escalate bind rbac

### PR DESCRIPTION
**Pull request by zhiweiyin318**

### CODE REVIEW

File: deploy/cluster-manager/config/rbac/cluster_role.yaml [hunk 1]
```diff
@@ -55,3 +58,65 @@ rules:
...
+  verbs: ["renew"]
+- apiGroups: ["work.open-cluster-management.io"]
+  resources: ["manifestworkreplicasets"]
+  verbs: ["get", "list", "watch", "update", "delete", "deletecollection", "patch"]
+- apiGroups: ["work.open-cluster-management.io"]
+  resources: ["manifestworkreplicasets/finalizers"]
+  verbs: ["update"]
+- apiGroups: ["work.open-cluster-management.io"]
+  resources: ["manifestworks"]
+  verbs: ["get", "list", "watch", "create", "update", "delete", "deletecollection", "patch", "execute-as"]
```
Comment (qiujian16) [inline deploy/cluster-manager/config/rbac/cluster_role.yaml:126]: also need create verb for manifestworkreplicaset controller

File: deploy/cluster-manager/config/rbac/cluster_role.yaml [hunk 2]
```diff
@@ -55,3 +58,65 @@ rules:
 - apiGroups: ["migration.k8s.io"]
   resources: ["storageversionmigrations"]
   verbs: ["create", "get", "list", "update", "watch", "patch", "delete"]
+
+- apiGroups: ["addon.open-cluster-management.io"]
+  resources: ["managedclusteraddons"]
+  verbs: ["get", "list", "watch"]
```
Comment (qiujian16) [inline deploy/cluster-manager/config/rbac/cluster_role.yaml]: create/update/patch is also need for addon-manager controller

File: deploy/cluster-manager/config/rbac/cluster_role.yaml (hunk 1 above)
Comment (zhiweiyin318) [reply deploy/cluster-manager/config/rbac/cluster_role.yaml:126]: added

File: deploy/cluster-manager/config/rbac/cluster_role.yaml (hunk 2 above)
Comment (zhiweiyin318) [reply deploy/cluster-manager/config/rbac/cluster_role.yaml]: added

**Pull request merged**